.Gruntfile.babel.js

```diff
@@ -208,7 +233,7 @@ module.exports = (grunt) => {
     }
   })
 
-  grunt.registerTask('backend:relaunch', '[internal]', function () {
+  grunt.registerTask('backend:relaunch', function () {
```

Comment:
Why remove the comment? It's there to indicate this shouldn't be called directly from the command line.

Reply:
is `"[internal]"` a comment? For what I know about grunt, the 2nd parameter of `registerTask ` should call other tasks and I didn't see any task called internal, so that's why I removed it because I thought it was stale code.